tests/build_/test_build_request.py

```diff
@@ -171,12 +171,9 @@ def assert_koji_upload_plugin(self, plugins, use_auth, valid=True):
     @pytest.mark.parametrize('kojihub', ("http://hub/", None))
     @pytest.mark.parametrize('use_auth', (True, False, None))
     def test_render_koji_upload(self, use_auth, kojihub):
-        enable = {
-            "plugin_type": "postbuild_plugins",
-            "plugin_name": "koji_upload",
-            "plugin_args": {},
-        }
-        build_request = BuildRequest(INPUTS_PATH)
+        inner_template = WORKER_INNER_TEMPLATE.format(
+            arrangement_version=DEFAULT_ARRANGEMENT_VERSION)
+        build_request = BuildRequest(INPUTS_PATH, inner_template=inner_template)
```

Comment:
This will mix the worker inner_template with the prod outer_template and customize_conf.
Maybe set all three?